databend_py/client.py

```diff
@@ -268,8 +269,27 @@ def upload(self, file_name, table_name, file_type=None):
                 file_type = file_name.split(".")[1].upper()
             else:
                 file_type = "CSV"
-        file_data = self.get_file_data(file_name)
-        self.sync_csv_file_into_table(file_name, file_data, table_name, file_type)
+        self.sync_csv_file_into_table(file_descriptor, file_name, table_name, file_type)
+
+    def upload_to_stage(self, file_descriptor, stage_path=None, file_name=None):
```

Comment:
the two parameters `stage_path` and `file_name` can be merged into one?

Reply:
IMO ,If no file_name param, we need make a random file name for it and it's not clear for user?

Reply:
The two parameters are reasonable and allow users to customize the Stage